modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```
@@ -268,6 +268,8 @@ action.VIEW_COMMERCE_NOTIFICATION_QUEUE_ENTRIES=View Notification Queue Entries
 action.VIEW_COMMERCE_NOTIFICATION_TEMPLATES=View Notification Templates
 action.VIEW_COMMERCE_ORDER_TYPES=View Commerce Order Types
 action.VIEW_COMMERCE_ORDERS=View Orders
+action.VIEW_COMMERCE_PRODUCT_ATTACHMENTS=View Attachments
+action.VIEW_COMMERCE_PRODUCT_IMAGES=VIew Images
```

@andrea-ale-sbarra typo here
```suggestion
action.VIEW_COMMERCE_PRODUCT_IMAGES=View Images
```